fuel-tx/src/tests/offset.rs

```diff
@@ -93,6 +93,17 @@ fn common_parts_create_and_script<Tx: Buildable>(tx: &Tx, bytes: &[u8], cases: &
             assert_eq!(predicate_data, predicate_data_p);
         }
 
+        if let Some(predicate_gas_used) = i.input_predicate_data() {
+            cases.predicate_data_coin = cases.predicate_data_coin || i.is_coin() && !predicate_gas_used.is_empty();
+            cases.predicate_data_message =
+                cases.predicate_data_message || i.is_message() && !predicate_gas_used.is_empty();
+
+            let ofs = input_ofs + i.predicate_data_offset().expect("input contains predicate data");
```

Comment:
This is pointing to the same offset as above. That seems wrong?

Reply:
The function `input_predicate_data` does not return `predicate_gas_used`

Reply:
this was mistakenly implemented as I added the predicate_gas_used field, it has been removed, I also noticed that the GTF was not fully implemented for predicate_gas_used but now it is